providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -265,6 +268,16 @@ def get_conn(self) -> client.ApiClient:
                 )
             return client.ApiClient()
 
+        if self.config_dict:
```

Comment:
should this new option self.config_dict be added here to avoid multiple setup of the configuration? https://github.com/VladaZakharova/airflow/pull/212/files#diff-99291d7e830bdd58d662b11fb9aea8e76753470b34ef2dcc8116e4be28a21c78R219

Reply:
@Crowiant it is a good point I will update the code